pyanaconda/modules/common/structures/policy.py

```diff
@@ -0,0 +1,139 @@
+#
+# DBus structures for the password policy.
+#
+# Copyright (C) 2020  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+from dasbus.structure import DBusData
+from dasbus.typing import *  # pylint: disable=wildcard-import
+
+from pyanaconda.anaconda_loggers import get_module_logger
+from pyanaconda.core.configuration.anaconda import conf
+
+log = get_module_logger(__name__)
+
+__all__ = ["PasswordPolicy"]
+
+
+class PasswordPolicy(DBusData):
+    """The password policy data."""
+
+    def __init__(self):
+        self._quality = UInt16(0)
+        self._length = UInt16(0)
+        self._empty = True
+        self._strict = False
+
+    @property
+    def quality(self) -> UInt16:
+        """The minimal quality of the password.
```

Comment:
I think he we should also mention that quality in this case means the `libpwquality` defined password quality score.

Reply:
Fixed.